### DISCUSSION TITLE
Due to overwhelmingly one-sided media coverage orchestrated by the GF and Debian Community Team, no one knows we won.

### DISCUSSION THREAD

**Discussion by cmpunches** [rms-support-letter/rms-support-letter.github.io#4741] [General]
Despite the fact that we got a great deal more signatures than the RMS open letter, the news is saturated with doom and gloom and defeat stories about the FSF instead of reporting on a resurgently thriving organization, the FSF, who is celebrating an affirmative win against shady corporate interests and their defamatory surrounding PR campaign.

We won.  We didn't just win, we won hard.  This so rarely happens!

I said repeatedly that we needed media coverage to folks or their PR team would just suppress awareness that this took place, because they're liars.

Well, here we are.  The media is reporting on an open letter as if it represented the f/oss community and was remotely impactful, and that same media is not at all reporting on the support letter here which got far more community support.  These people aren't corporate stooges looking to virtue signal for a career hop -- these people took a real risk to sign for something they wanted to stand up to-- and it worked!

It's not a win if nobody knows you won.

The only way I can see a persistence of our victory is by continuing the letter's signature window to just keep building the numbers up, and getting news of it on the various blogs in the open source community.  Get folks talking about it on their podcasts.  Some kind of organization really, really needs to happen there.

Sure, the news is misrepresenting the state of the open source community and the FSF.  Because we're not marketing/promoting to media that this happened, so their literal only input from the commmunity is coming from the Gnome Foundation / Debian Group that ran the very smear campaign that we completely crushed -- and trust me, Neil McGovern is desperately playing damage control along with Elana Hashman and Molly de Blanc, because this fell apart hard on them and they need to work the narrative to squeeze through this.  If it goes on too long, the media might become of a general consensus that their having been conned by the shelled out OSI ex-board member is too much of an embarrassment to correct given the FSF's relatively pacifist stance.

**Comment by ghost** [comment]
No one needs to know we won, FSF already made their decision to keep RMS, the Twitter mobs will continue to seethe and we will continue to give them zero attention except reporting them to the police for targeted harassment. Let them take the L.

**Comment by rldmt** [comment]
It's funny how engaged googlers
https://gcc.gnu.org/pipermail/gcc/2021-March/235147.html

redhaters
https://gcc.gnu.org/pipermail/gcc/2021-March/235094.html


and people from suse 
https://gcc.gnu.org/pipermail/gcc/2021-March/235108.html

are in this debate against stallman.

**Comment by ghost** [comment]
@rldmt Fourteen years ago, Penn and Teller made this https://youtu.be/yi3erdgVVTw.
I am amazed at how relevant it is now, fourteen years later (that's fourteen, FOURTEEN!) in the free software community.

**Comment by Bluesboy** [comment]
Very inspiring, returned my belief in humankind.

**Comment by franpoli** [comment]
We won already? I am almost disappointed. I was about to commit a new translation.

**Reply by shenlebantongying** [reply]
I personally think we don't win,  but we also didn't lose for now.

**Reply by Bluesboy** [reply]
Proceed with translation. As TS stated, we should keep signing until it's be clear that we crushed them.

**Reply by cmpunches** [reply]
yes please do, @franpoli this is far from over.  and please distribute links to people who can read that translation too!

**Reply by jobbautista9** [reply]
What, you want us to lose? :P

As others said above, please commit a new translation. We need more international support and solidarity! :)

**Comment by 6r1d** [comment]
Lol, a related discussion: https://github.com/rms-support-letter/rms-support-letter.github.io/discussions/6471
